### DISCUSSION TITLE
.NET no WSL

### DISCUSSION THREAD

**Discussion by DrenokBR** [balta-io/forum#445] [Tira Dúvidas] [answered]
Fala André !!!

Cara, Eu uso Windows. Já vários tutoriais de instalação do .NET no Ubuntu (WSL)... faço todos os passos, mas não estou conseguindo instalar de jeito nenhum. Poderia me ajudar? Tem algum link com um tutorial mais confiável ? Ou pode me dar alguma dica ?

**Comment by DrenokBR** [comment]
O comando "dotnet --info" funciona, mas o "dotnet --version" não funciona.

**Reply by DrenokBR** [reply]
![dotnetNaoFuncionaNoWSL](https://github.com/balta-io/forum/assets/36772075/469e1493-4438-4eed-99c9-c8dae6a9b8ac)

**Reply by DrenokBR** [reply]
![dotnetNaoFuncionaNoWSL2](https://github.com/balta-io/forum/assets/36772075/19323e22-f257-4308-9ab1-2bd6a804eae2)

**Reply by andrebaltieri** [reply, marked as answer]
Boa tarde, @DrenokBR, como vai?

No caso, me parece que você instalou o **RUNTIME** do .NET e não a **SDK**.
https://learn.microsoft.com/en-us/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website

Em adicional, no Windows eu não instalo o .NET no WSL, uso ele no Windows mesmo, só uso o Docker no WSL, acho bem mais simples.
Mas de qualquer forma, seguindo a instalação do SDK, deve funcionar.

**Reply by DrenokBR** [reply]
Pode ignorar... só fui corrigir a palavra Ubuntu que estava como "Ubunto" (rsrs...).
Não sabia que o ato de editar jogava a "Discussion" para o topo... (rsrs...)
